tests/native_api_tests/test_api_model.cpp

```diff
@@ -1099,9 +1100,10 @@ TEST_CASE("API Model") {
         for (auto const method : {PGM_default_method, PGM_iterative_linear, PGM_newton_raphson}) {
             CAPTURE(method);
             if (method == PGM_newton_raphson) {
-                CHECK_THROWS_WITH_AS(run_se_with_current_sensor(method, PGM_experimental_features_disabled),
-                                     "Newton-Raphson state estimation is not implemented for current sensors",
-                                     PowerGridRegularError);
+                CHECK_THROWS_WITH_AS(
+                    run_se_with_current_sensor(method, PGM_experimental_features_disabled),
+                    "Newton-Raphson state estimation is not implemented for global angle current sensors",
+                    PowerGridRegularError);
```

Comment:
optional: maybe good to also loop to check that it is actually not experimental if you do have a local current sensor with NRSE but only if you really have a global one